### ISSUE TITLE
prepare-commit-msg hook failure not always open "Git Error" window

### ISSUE THREAD

**Issue by flaviorighi** [ForkIssues/TrackerWin#1413]
When prepare-commit-msg hook fails because the commit message template is not fullfilled correctly, the Git Error window is not opened most of the times.

In my hook I only check if the user replaces the commented lines.
Here is my hook:

```
if  grep -q '^#' $1 
then
    echo "error" 1>&2
    exit 1
fi
exit 0
```

I've tried without redirection to stderr, but the window is opened only sometimes.

**Comment by DanPristupov** [comment]
It works for me and the windows always appear. I'll keep the hook for now, may be I'll reproduce the problem.

> I've tried without redirection to stderr, but the window is opened only sometimes.

In the majority of cases Fork recognizes errors by exit code (0 == success).

**Comment by flaviorighi** [comment]
After some other tries I've found that the issue only happens if the template is unmodified.

I'll explain more what happens:
The click on the "Title" or "description" field fills the fields with the template content.
If I press commit without edit texts, the error window not appears.
If one of two fields are edited, then it works correctly.

I've noticed because I was doing some tests on my hooks, I know that this probably never happen in a real use case.
